lib/zendesk_apps_support/validations/manifest.rb

```diff
@@ -270,8 +270,10 @@ def invalid_v1_location(package)
           end
         end
 
-        def invalid_location_uri_error(package, path)
-          return nil if path == ZendeskAppsSupport::Manifest::LEGACY_URI_STUB
+        def invalid_location_uri_error(package, location_options)
+          path = location_options.url
+          return nil if path == ZendeskAppsSupport::Manifest::LEGACY_URI_STUB ||
+                        (!location_options.signed && !path.index('{{setting.').nil?)
```

Comment:
Could we return a more descriptive error message if the url is signed and has a setting placeholder?